packages/client/src/components/app/forms/RelationshipField.svelte

```diff
@@ -139,14 +147,14 @@
     filter: UISearchFilter | undefined,
     linkedTableId?: string
   ) => {
-    const datasource =
+    const datasource: DataFetchDatasource =
       dsType === "table"
         ? {
             type: dsType,
             tableId: linkedTableId!,
           }
         : {
-            type: dsType,
+            type: appUsersOnly ? "table" : dsType,
```

Comment:
As long as pass `appUsersOnly` in to `createFetch` here as a param, it'll be reactive.